### PR TITLE
[PLATFORM-1228] Improve the layout of Set Price section for standard products

### PR DESCRIPTION
**Pull request by juhah**
This adds different layout for standard products price setting:
Screen Shot 2020-02-05 at 21 25 31

Design: https://share.goabstract.com/53ce3bc8-b8fb-49c8-9859-84c0485e96f0?mode=design&sha=8e5e8db7de798d4970de1594ea92ea8f22776f34

Data union product remains the same:
Screen Shot 2020-02-05 at 21 25 10

Design: https://share.goabstract.com/f110e1ef-fad8-481d-ba62-5c3f27175290?mode=design&sha=8e5e8db7de798d4970de1594ea92ea8f22776f34

### CODE REVIEW

File: app/src/app/index.jsx [hunk 1]
```diff
@@ -172,7 +172,7 @@ const MarketplaceRouter = () => (process.env.DATA_UNIONS ? [
     <Route exact path={formatPath(marketplace.products, ':id', 'publish')} component={ProductPublishPage} key="ProductPublishPage" />,
     <Route exact path={formatPath(marketplace.products, ':id', 'streamPreview', ':streamId')} component={StreamPreviewPage} key="StreamPreview" />,
     <Route exact path={formatPath(marketplace.products, ':id')} component={ProductPage} key="ProductPage" />,
-    <Route exact path={routes.editProduct()} component={EditProductAuth} key="EditProduct" />,
+    <Route exact path={formatPath(marketplace.products, ':id', 'edit')} component={EditProductAuth} key="EditProduct" />,
```
Comment (juhah) [inline app/src/app/index.jsx]: Fix for edit route not working when `DATA_UNIONS` flag is not defined in `.env`

File: app/src/marketplace/containers/EditProductPage/BeneficiaryAddress.jsx [hunk 2]
```diff
@@ -146,9 +146,11 @@ const BeneficiaryAddress = ({
                 htmlFor="beneficiaryAddress"
                 className={cx(styles.root, styles.BeneficiaryAddress, className)}
             >
-                <strong>
-                    <Translate value="editProductPage.setPrice.setRecipientEthAddress" />
-                </strong>
+                <Translate
+                    value="editProductPage.setPrice.setRecipientEthAddress"
+                    tag="div"
+                    className={styles.label}
```
Comment (mondoreale) [inline app/src/marketplace/containers/EditProductPage/BeneficiaryAddress.jsx]: Hm, this class shouldn't exists. 🤔 Let's avoid it with this:
```jsx
import Label from '$ui/Label'
/* … */
<Label
  as={Translate}
  value="editProductPage.setPrice.setRecipientEthAddress"
  tag="div"
/>
```
💃

File: app/src/marketplace/containers/EditProductPage/beneficiaryAddress.pcss [hunk 3]
```diff
@@ -1,15 +1,12 @@
...
-  align-items: center;
-
-  strong {
-    font-weight: var(--medium);
-  }
+  width: 100%;
 }
 
 .BeneficiaryAddress {}
+
```
Comment (mondoreale) [inline app/src/marketplace/containers/EditProductPage/beneficiaryAddress.pcss]: After using `$ui/Label` this can be removed. `#lesscodelessmess`

**Comment by fonty1** [comment]
@mattatgit in:
https://app.abstract.com/projects/7b5a5630-4532-11e9-8209-7d06014a7ce5/branches/master/commits/8e5e8db7de798d4970de1594ea92ea8f22776f34/files/9DA01BE6-D016-4BCC-AB08-F3480AF49F15/layers/01F7DAF9-13F1-49D4-AA7C-CFB061225177?mode=build&selected=775952580-C223E043-A666-43CF-A27D-C56307198918
there are lots of mentions of 'product', i.e. "Set the price for your product here". Should these be replaced with "Data Union"?

@juhah LGTM - only thing I noticed was "Fix price in fiat for protection against shifts in the DATA price" should be `font-weight: 500`

**Comment by juhah** [comment]
> "Fix price in fiat for protection against shifts in the DATA price" should be font-weight: 500

Thanks, fixed in https://github.com/streamr-dev/streamr-platform/pull/866/commits/567931e0a6bdd140c0a16918f8dff535bb8fccf9:

Screen Shot 2020-02-07 at 14 34 25

@mondoreale Fixed the label component also. It's centered enough for my eyes so I'll leave it

Screen Shot 2020-02-07 at 14 34 05

**Comment by mattatgit** [comment]
@juhah Maybe just check the vert alignment of the help tip "?" icon - looks a bit low of vertical centre of that line.

**Comment by juhah** [comment]
Adjusted the alignment:
Screen Shot 2020-02-10 at 10 16 11

**Pull request merged**